events/code_review_events/workflow.py

```diff
@@ -82,6 +98,16 @@ def __init__(
             self.user_blacklist = {}
             logger.info("No blacklisted user")
 
+        # A map from try push task group to its linked Phabricator build.
+        self.task_group_to_build = {}
+        # A map from build phid to try revision.
+        self.diff_to_push = {}
+
+        # TODO: Use asnyc version of these services.
```

Comment:
The official TaskclusterConfig lib supports that : `taskcluster_config.get_service("index", use_async=True)`

Reply:
Yep, but we don't have it yet, and this would likely require a larger change, so we can do that in a follow-up.

Reply:
I filed https://github.com/mozilla/code-review/issues/268, and I'll remove the TODO.